modules/virt-adding-vtpm-to-vm.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-using-vtpm-devices.adoc
+
+:_content-type: PROCEDURE
+[id="virt-adding-vtpm-to-vm_{context}"]
+= Adding a vTPM device to a virtual machine
+
+Adding a virtual Technical Platform Module (vTPM) device to a virtual machine
+(VM) allows you to run a VM created from a Windows 11 image without a physical
+TPM device. A vTPM device also temporarily stores secrets for that VM.
+
+.Procedure
+
+. Open the VM manifest in your default editor by running the following command:
++
+[source,terminal]
+----
+$ oc edit vm <vm_name>
+----
+
+. Edit the VM `spec` so that it includes the `tpm: {}` line. For example:
++
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+    name: example-vm
+spec:
+  template:
+    spec:
+      domain:
+        devices:
+          tpm: {} <1>
+...
+----
+<1> Adds the TPM device to the VM.
+
+. Apply the configuration to the cluster by running the following command:
```

Comment:
Remove this step. No apply manifest config is needed if VM config is updated via oc edit